graphitron-codegen-parent/graphitron-java-codegen/src/main/java/no/sikt/graphitron/generators/mapping/RecordMapperMethodGenerator.java

```diff
@@ -49,11 +50,17 @@ protected CodeBlock iterateRecords(MapperContext context) {
 
             var innerContext = context.iterateContext(innerField);
             var isType = innerContext.targetIsType();
-            var previousHadSource = innerContext.getPreviousContext().hasSourceName();
+            MapperContext previousContext = innerContext.getPreviousContext();
+            var previousHadSource = previousContext.hasSourceName();
 
             var innerCode = CodeBlock.builder();
             if (innerField.isResolver()) {
-                innerCode.add(innerContext.getResolverKeySetMappingBlockForJooqRecord());
+                if (innerContext.hasTable() && innerField.isIterableWrapped()
+                        && !previousContext.getTarget().isIterableWrapped()) {
```

Comment:
```suggestion
                if (processedSchema.isOrderedMultiKeyQuery(field)) {
```
Fungerer noe sånt mon tro? 🤔